terraform/builder-monolith.tf

```diff
@@ -20,6 +20,21 @@ resource "aws_instance" "monolith" {
         agent       = "${var.connection_agent}"
     }
 
+    ebs_block_device {
```

Comment:
I don't think this is what we actually want to do here. This will define an ebs volume for us but it's lifecycle is part of the instance. I think the route is to rather rather an ebs volume separately and attach it with these resouces:

* https://www.terraform.io/docs/providers/aws/r/ebs_volume.html
* https://www.terraform.io/docs/providers/aws/r/volume_attachment.html

I believe this is the correct approach when you need a persistent file system regardless of the instance owning it or not